### PR TITLE
Adding notes to the 21th&5th episodes

### PR DESCRIPTION
**Pull request by mohamedaitbouaaza**
@yjose

### CODE REVIEW

File: blablas/ep21/index.md [hunk 1]
```diff
@@ -13,14 +13,58 @@ audio:
...
+
+00:40 - The importance of extracurricular activities in academic and professional careers.
+
+00:50 - Is that the exams score is quite important ?
+
+00:58 - How to find a good end-of-school project (PFE) and prepare for the professional world ?
+
+01:11 - End-of-school project (PFE) in a multinational company Vs Startup.
+
+01:19 - How to prepare for a good defense ?
```
Comment (yjose) [inline blablas/ep21/index.md:48]: Not sure what you mean by `good defense`

File: blablas/ep5/index.md [hunk 2]
```diff
@@ -3,13 +3,39 @@ date: 2018-12-19
...
-[//]: # "Check this github issue on How to add Episode Notes  https://github.com/DevC-Casa/geeksblabla.com/issues/23 "
+In this episode of GeeksBlabla MSS, we welcome Taher Alami :Founder & CEO of @Abweb , to talk about his journey in IT field.
 
-...
+## Guests
+
+- [Taher Alami](https://twitter.com/taheralami)
+
+## Notes
+
```
Comment (yjose) [inline blablas/ep5/index.md:21]: Not sure if you worked on Taher El Alami episode if it's the case,  the Notes and links didn't seem to be right?

File: blablas/ep21/index.md (hunk 1 above)
Comment (mohamedaitbouaaza) [reply blablas/ep21/index.md:48]: its mean project presentation "soutenance"

File: blablas/ep5/index.md (hunk 2 above)
Comment (mohamedaitbouaaza) [reply blablas/ep5/index.md:21]: Actually, I didn't pay attention that I include the Taher El Alami episode changes too in this PR, but I will work on it separately to add the Notes and links

File: blablas/ep21/index.md (hunk 1 above)
Comment (yjose) [reply blablas/ep21/index.md:48]: Hmm, a little confusing, why not `good project presentation `

File: blablas/ep5/index.md (hunk 2 above)
Comment (yjose) [reply blablas/ep5/index.md:21]: ok Great, please mention in the #23  issue that you are working on it, and I would love if you can add it to this PR

File: blablas/ep21/index.md (hunk 1 above)
Comment (mohamedaitbouaaza) [reply blablas/ep21/index.md:48]: Yeah, I will modify it

File: blablas/ep5/index.md (hunk 2 above)
Comment (mohamedaitbouaaza) [reply blablas/ep5/index.md:21]: ok, I will . thanks for your review

File: blablas/ep21/index.md [hunk 3]
```diff
@@ -13,14 +13,58 @@ audio:
...
 ## Notes
 
-<br/>
-<br/>
-<br/>
+00:00 - Introduction: Welcoming and presenting guests.
+
+00:08 - First year of school experience and some tips.
+
+00:28 - Tips to deal better with the coming years of school and The importance of self-learning.
```
Comment (yjose) [inline blablas/ep21/index.md:34]: Tips to deal with  upcoming years

File: blablas/ep21/index.md [hunk 4]
```diff
@@ -13,14 +13,58 @@ audio:
...
+
+00:36 - Is that the degree is important to integrate the field of development ?
```
Comment (yjose) [inline blablas/ep21/index.md:36]: The importance of school degrees in the development field?

File: blablas/ep21/index.md [hunk 5]
```diff
@@ -13,14 +13,58 @@ audio:
...
+
+00:38 - Freelancing while studying.
+
+00:40 - The importance of extracurricular activities in academic and professional careers.
+
+00:50 - Is that the exams score is quite important ?
```
Comment (yjose) [inline blablas/ep21/index.md:42]: Are exams scores quite important for your professional careers  ?

File: blablas/ep5/index.md [hunk 6]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+00:00 - Introduction: Welcoming and presenting the guest.
+
+00:10 - Taher Alami debut in the web.
```
Comment (yjose) [inline blablas/ep5/index.md:24]: Taher Alami's first years in web development.

File: blablas/ep5/index.md [hunk 7]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+00:15 - First experience with entrepreneurship.
+
+00:18 - Is anyone can develop? and How to be successful as a web developer?
```
Comment (yjose) [inline blablas/ep5/index.md:28]: Can anyone become a software developer? ....

File: blablas/ep5/index.md [hunk 8]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+00:22 - Future of the Moroccan web.
+
+00:25 - The Moroccan web community and its future.
+
+00:29 - Working as a developer in Dubai.
+
+00:32 - Startup ecosystem in Morocco.
+
+00:40 - Is it easier to begin a startup in Dubai than Morocco?
```
Comment (yjose) [inline blablas/ep5/index.md:38]: Is it easier to start a ......?

File: blablas/ep5/index.md [hunk 9]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+00:41 - Single or multiple technology learning?
```
Comment (yjose) [inline blablas/ep5/index.md:40]: focus on learning one technology or multiple at once?

File: blablas/ep5/index.md [hunk 10]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+00:49 - Why should learn about blockchain as a freenalcer?
+
+00:52 - Invesstissement in Africa and the experience of Taher Alami in it.
```
Comment (yjose) [inline blablas/ep5/index.md:44]: Investment in Africa and Taher Alami experience in this field.

File: blablas/ep5/index.md [hunk 11]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+00:59 - Big data and cloud computing in morocco.
```
Comment (yjose) [inline blablas/ep5/index.md:46]: Morocco

File: blablas/ep5/index.md [hunk 12]
```diff
@@ -3,13 +3,70 @@ date: 2018-12-19
...
+
+01:02 - AR and Spark AR topic.
```
Comment (yjose) [inline blablas/ep5/index.md:48]: AR and Spark AR.

**Comment by yjose** [comment]
@all-contributors please add @mohamedaitbouaaza  for content

**Pull request merged**